src/config/mod.rs

```diff
@@ -4651,7 +4652,11 @@ fn to_change_value(v: &str, typed: &ConfigValue) -> CfgResult<ConfigValue> {
         ConfigValue::Usize(_) => ConfigValue::from(v.parse::<usize>()?),
         ConfigValue::Bool(_) => ConfigValue::from(v.parse::<bool>()?),
         ConfigValue::String(_) => ConfigValue::String(v.to_owned()),
-        _ => unreachable!(),
+        ConfigValue::Schedule(_) => {
+            let schedule = v.parse::<ReadableSchedule>()?;
+            ConfigValue::from(schedule)
+        }
+        ConfigValue::Skip | ConfigValue::None | ConfigValue::Module(_) => unreachable!(),
```

Comment:
How about logging  and returning an error?

Reply:
ConfigValue::Skip | ConfigValue::None | ConfigValue::Module(_) will and should not be reached. I think it's fine as unlike the previous `_ => unreachable!()`, which may forget to implement some type, it is intentional.